packages/block-library/src/image/edit.js

```diff
@@ -404,10 +405,12 @@ export class ImageEdit extends Component {
 				src={ url }
 			/>
 		);
+		const needsAlignmentWrapper =
+			[ 'center', 'left', 'right' ].indexOf( align ) !== -1;
```

Comment:
This could be simplified to:
```suggestion
			[ 'center', 'left', 'right' ].includes( align );
```